peer.go

```diff
@@ -1249,7 +1249,7 @@ func (p *peer) writeMessage(msg lnwire.Message) error {
 	n, err := lnwire.WriteMessage(b, msg, 0)
 	atomic.AddUint64(&p.bytesSent, uint64(n))
 
-	// TODO(roasbeef): add write deadline?
+	p.conn.SetWriteDeadline(time.Now().Add(10 * time.Second))
```

Comment:
Can you lift this (`10 * time.Second`) into a constant at the top of the file? Thanks!

Reply:
done